src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Status.tsx

```diff
@@ -61,78 +64,70 @@ export const Status = () => {
     browserHistory.goBack()
   }
 
-  const { statusIcon, statusText, statusBg } = (() => {
-    let statusIcon: React.ReactElement = <ArrowForwardIcon />
+  const { statusIcon, statusText, statusBg, statusBody } = (() => {
+    let statusIcon: React.ReactElement = <AssetIcon size='xs' src={asset?.icon} />
     let statusText = StatusTextEnum.pending
-    let statusBg = defaultStatusBg
+    let statusBg = 'transparent'
+    let statusBody = translate('modals.withdraw.status.pending')
     if (state.withdraw.txStatus === 'success') {
       statusText = StatusTextEnum.success
-      statusIcon = <CheckIcon color='green' />
+      statusIcon = <CheckIcon color='white' />
       statusBg = 'green.500'
+      statusBody = translate('modals.withdraw.status.success', {
+        opportunity: `${underlyingAsset.symbol} Vault`,
+      })
     }
     if (state.withdraw.txStatus === 'failed') {
       statusText = StatusTextEnum.failed
-      statusIcon = <CloseIcon color='red' />
+      statusIcon = <CloseIcon color='white' />
       statusBg = 'red.500'
+      statusBody = translate('modals.withdraw.status.failed')
     }
 
-    return { statusIcon, statusText, statusBg }
+    return { statusIcon, statusText, statusBg, statusBody }
   })()
 
   return (
     <TxStatus
       onClose={handleCancel}
-      onContinue={handleViewPosition}
-      loading={state.loading}
+      onContinue={state.withdraw.txStatus === 'success' ? handleViewPosition : undefined}
+      loading={state.withdraw.txStatus !== ('success' || 'failed ')}
```

Comment:
```
> 'failed' !== ('success' || 'failed ')
> true
```

this doesn't work like you're intending to, you might want

`!['success', 'failed'].includes(state.withdraw.txStatus)`